build.gradle.kts

```diff
@@ -3,6 +3,10 @@ buildscript {
         google()
         mavenCentral()
     }
+    dependencies {
+        classpath(Libraries.Hilt.gradlePlugin)
+        classpath("org.jetbrains.kotlin:kotlin-gradle-plugin:1.6.10")
```

Comment:
I don't think this line is necessary, as it's already being added in `buildSrc/build.gradle.kts`

We can upgrade the version tho. From `1.5.31`, to `1.6.10`.

Perhaps moving `Hilt.gradlePlugin` there too.